src/pages/EmployeeProfile/components/BannerItem.jsx

```diff
@@ -0,0 +1,24 @@
+//import
+import { Avatar, Paper, Stack } from "@mui/material";
+import "./style.css"
+
+import Image from '../media/plant-banner.jpg'
+
+export default function BannerItem() {
+
+    const styles = {
+        paperContainer: {
+            backgroundImage: `url(${Image})`
+        }
+    }
+
+    return (
+        <div className="banner">
+            <Paper style={styles.paperContainer} className="banner-paper">
```

Comment:
Shouldnt be stlyed with rounded corners to avoid the "white parts" in the corners:
![image](https://github.com/SE2-WS23/Skillverse-Web/assets/65715000/bffd220d-3e92-4ec7-be24-dd459d4cb183)